docs/src/content/index.module.css

```diff
@@ -37,7 +42,7 @@
   background-image: var(--robotImg);
   overflow: hidden;
   height: 0;
-  padding-top: calc(656/960*100%);
+  padding-top: calc(656 / 960 * 100%);
```

Comment:
We made a change here pairing to add the vars which must not have made it into the commit.

Reply:
File saved and committed. 😄